src/bp-core/bp-core-filters.php

```diff
@@ -1417,3 +1417,20 @@ function buddyboss_platform_plugin_update_notice() {
 	}
 	add_action( 'admin_notices', 'buddyboss_platform_plugin_update_notice' );
 }
+
+/**
+ * Set the default value checked "Show Avatars" in Settings > Discussion > Avatars > Avatar Display.
+ *
+ * @param $value
+ * @param $option
+ * @param $default
+ *
+ * @return mixed|void
+ *
+ * @since BuddyBoss 1.4.8
+ */
+function bp_core_set_default_show_avatars( $value, $option, $default ) {
+	$value = true;
+	return apply_filters( 'bp_core_set_default_show_avatars', $value, $option, $default );
```

Comment:
@chetansatasiya can you add phodoc for the new filter provided.

Reply:
@mehulkaklotar added the doc